src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -27,9 +27,25 @@
 from turicreate.toolkits._internal_utils import (_raise_error_if_not_sframe,
                                                  _numeric_param_check_range)
 
-def create(dataset, target, feature = None, model = 'resnet-50',
-           validation_set='auto', max_iterations = 10, verbose = True,
-           seed = None, batch_size=64):
+_DEFAULT_SOLVER_OPTIONS = {
+'convergence_threshold': 1e-2,
+'step_size': 1.0,
+'lbfgs_memory_level': 11,
+'max_iterations': 10}
+
+def create(dataset, target, features=None, model = 'resnet-50',
```

Comment:
The `feature` parameter has been renamed to `features` which breaks things in the body of the function. Please keep the names of the existing parameter the same.

Reply:
@TobyRoseman I must not have caught that when I modelled the code off of logistic_regression.py. It's fixed now.

Reply:
@TobyRoseman I must not have caught that when I modelled the code off of logistic_classifier.py. It's fixed now.

Reply:
Sorry for all the repeated back and forth my errors are causing. Are there other corrections that need to be made?

Reply:
It looks good to me, but right now we don't know if additional changes are needed. The Travis CI build is broken for unrelated reasons. We're investigating that in #1330. Once the build is fixed, we can test these changes to find out if anything else is needed.

Reply:
Good to know.

Reply:
Looking at #1330, it seems like Travis CI is working again and this latest commit can be tested.

Reply:
@varunlakshmanan - please rebase your changes on top of master and push, so these changes can be tested.